AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DisplayMapperAdded.py

```diff
@@ -129,52 +188,233 @@ def AtomEditorComponents_DisplayMapper_AddedToEntity():
+                AtomComponentProperties.display_mapper('Max Point (luminance)'), value=1005.7191162)
+
+            # 17. 'Surround Gamma' set high and low value
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Surround Gamma'), value=1.2)
+            Report.result(Tests.surround_gamma, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Surround Gamma')), 1.2, TOLERANCE))
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Surround Gamma'), value=0.6)
+            Report.result(Tests.surround_gamma, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Surround Gamma')), 0.6, TOLERANCE))
+
+            # 18. 'Gamma' set high and low value
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Gamma'), value=4.0)
+            Report.result(Tests.gamma, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Gamma')), 4.0, TOLERANCE))
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Gamma'), value=0.2)
+            Report.result(Tests.gamma, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Gamma')), 0.2, TOLERANCE))
+
+            # 19. Enter/Exit game mode.
+            TestHelper.enter_game_mode(Tests.enter_game_mode)
+            general.idle_wait_frames(1)
+            TestHelper.exit_game_mode(Tests.exit_game_mode)
+
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Enable LDR color grading LUT'), False)
 
-        # 6. Set Enable LDR color grading LUT property True
         display_mapper_component.set_component_property_value(
             AtomComponentProperties.display_mapper('Enable LDR color grading LUT'), True)
-        Report.result(
-            Tests.enable_ldr_color_grading_lut,
-            display_mapper_component.get_component_property_value(
-                AtomComponentProperties.display_mapper('Enable LDR color grading LUT')) is True)
 
-        # 7. Enter/Exit game mode.
-        TestHelper.enter_game_mode(Tests.enter_game_mode)
-        general.idle_wait_frames(1)
-        TestHelper.exit_game_mode(Tests.exit_game_mode)
+        cinema_limit_white_presets = [48.0, 184.3200073, 368.6400146, 737.2800293]
+        for preset in DISPLAY_MAPPER_PRESET.keys():
+            # 20. Select and load each preset
+            # Preset Selection cannot be set or loaded currently; as a workaround we are calling an ebus to load preset
```

Comment:
do we have an issue cut to do this properly instead of with a workaround in the future? we should cut one if not